### DISCUSSION TITLE
Use cases

### DISCUSSION THREAD

**Discussion by mokhosh** [ryangjchandler/orbit#46] [General]
Hey Ryan!

Nice idea. I just assume this is not meant for projects where you have a ton of data that needs indexing and other advanced features.

Would love to hear your own thoughts about the appropriate use cases for this.

**Comment by ryangjchandler** [comment]
HI @mokhosh - in it's current state, Orbit is perfect for content management (blogs, portfolio sites) as well as smaller scale sites that are heavily content driven, think marketing sites, etc.

I would like to optimise the caching and insertion process of Orbit so that it's a viable option for all small apps, but to be honest, you can even use Orbit for a small part of your application too.

You can run the main, heavier part of your application through a MySQL database and then run the blog through Orbit without any issues.

**Reply by mokhosh** [reply]
That's great. I can see the benefits for spinning up a small website without any dependencies.

But when you already have a database, are there any pros in using Orbit for part of your application?

**Reply by ryangjchandler** [reply]
It all depends on workflow, personal preference and project type. I definitely think it's beneficial to have flat-files that power blogging, content heavy websites. The Git integration allows you to store data & content in version control and have the same data across environments.

**Reply by mokhosh** [reply]
thanks Ryan that makes sense
